src/main/java/com/github/virtualgamers/hrtg/rest/controller/core/PartyController.java

```diff
@@ -0,0 +1,48 @@
+package com.github.virtualgamers.hrtg.rest.controller.core;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import com.github.virtualgamers.hrtg.core.model.Party;
+import com.github.virtualgamers.hrtg.core.model.User;
+import com.github.virtualgamers.hrtg.core.service.PartyService;
+
+/**
+ * API for the {@link PartyService}
+ *
+ * @author CCThomas
+ *
+ */
+@RestController
+@RequestMapping("/party")
+public class PartyController {
+
+    Logger logger = LoggerFactory.getLogger(PartyController.class);
+
+    @Autowired
+    PartyService partyService;
+
+    @PostMapping("/addToParty/{partyId}/{username}")
```

Comment:
also, are we doing rest? bc then wouldn't we want plural nouns, eg?:
@POST /parties/{partyId}/add {usn in body}
@GET /parties/{partyId}
@POST /parties/start {usn in body}